beat/heart.py

```diff
@@ -1235,6 +1251,55 @@ class GeodeticResult(Object):
     processed_res = Array.T(shape=(None,), dtype=num.float, optional=True)
     llk = Float.T(default=0., optional=True)
 
+##Mahdi
+class PolarityTarget(gf.meta.Receiver):
+    '''
+    A seismogram computation request for a single component, including
+    its post-processing parmeters.
+    '''
+    codes = Tuple.T(
+        4, String.T(), default=('', 'STA', '', 'Z'),
+        help='network, station, location and channel codes to be set on '
+             'the response trace.')
+
+    elevation = Float.T(
+        default=0.0,
+        help='station surface elevation in [m]')
+
+    store_id = gf.meta.StringID.T(
+        optional=True,
+        help='ID of Green\'s function store to use for the computation. '
+             'If not given, the processor may use a system default.')
+
+    azimuth = Float.T(
+        optional=True,
+        help='azimuth of sensor component in [deg], clockwise from north. '
```

Comment:
here the azimuth deg vs rad mess needs cleaning

Reply:
standard pyrocko azimuth is in deg, so you could use azimuth_rad ..., likewise with take_offangles

Reply:
Done!

Reply:
Aren't they a little long? azimuth_rad, distance_deg, take_offangle_rad?

Reply:
lenght is not an issue its all about consistency and understandability